etc/ping.profile

```diff
@@ -41,7 +41,7 @@ private
 #private-bin has mammoth problems with execvp: "No such file or directory"
 private-dev
 # /etc/hosts is required in private-etc; however, just adding it to the list doesn't solve the problem!
-#private-etc resolv.conf,hosts,ca-certificates,ssl,pki,crypto-policies
+#private-etc alternatives,resolv.conf,hosts,ca-certificates,ssl,pki,crypto-policies
```

Comment:
Probably not needed in this profile. `ping` doesn't really depend on external utilities :)